packages/graphile-build-pg/src/plugins/makeProcField.js

```diff
@@ -190,10 +222,41 @@ export default function makeProcField(
       fieldScope.pgFieldIntrospectionTable = returnTypeTable;
       payloadTypeScope.pgIntrospectionTable = returnTypeTable;
     }
+  } else if (isRecordLike) {
+    const RecordType = getTypeByName(
+      inflection.functionReturnsRecordType(proc)
+    );
+    if (proc.returnsSet) {
+      if (isMutation) {
+        type = new GraphQLList(RecordType);
+      } else if (forceList) {
+        type = new GraphQLList(RecordType);
+        fieldScope.isPgFieldSimpleCollection = true;
+      } else {
+        const ConnectionType = getTypeByName(
+          inflection.scalarFunctionConnection(proc)
```

Comment:
I'm okay with using this method, but please add a comment to the inflectors saying it's now a misnomer and isn't just used for scalars.

Reply:
At this point, I have a good handle on the logic changes, so I propose we leave PgScalarFunctionConnectionPlugin.js/scalarFunctionEdge()/scalarFunctionEdge() unmodified by this PR and create a new PgRecordFunctionConnectionPlugin.js with recordFunctionEdge()/recordFunctionConnection() inflectors. Then there would no need to explain the hijacking and fewer sprinkles of `recordType.id === "2249"` throughout. Thoughts?

Reply:
Sure 👍

Reply:
Done. 10b9147